src/OpenLoco/Vehicles/Vehicle.h

```diff
@@ -130,14 +130,14 @@ namespace OpenLoco::Vehicles
             return as<VehicleBody, VehicleThingType::body_continued>();
         }
         bool isVehicle2Or6() { return is<VehicleThingType::vehicle_2>() || is<VehicleThingType::tail>(); }
-        vehicle_26* asVehicle2Or6() const
+        Vehicle2or6* asVehicle2Or6() const
```

Comment:
Inconsistent capitalisation between the function and the type. Personally, I prefer `Vehicle2or6` over `Vehicle2Or6`. The latter may be following the code style more strictly, but it looks a bit like `20r6`. I realise the type is likely to be renamed at a later point, though.

Reply:
It's likely going to be staying for a while as it's quite hard to work out which fields that are common that this is for. I did think about capitalising the or but felt it looked nicer without. Don't mind changing it.

Reply:
I don't feel strongly about going either way. Your call.